### DISCUSSION TITLE
ViTPose using a pretrained model.

### DISCUSSION THREAD

**Discussion by amartincirera** [open-mmlab/mmpose#2791] [General]
Hello,

I'm attempting to train a model using the VitPose architecture starting from a pretrained model and using a custom dataset with a skeleton and a different number of key points that used for human pose. I'm encountering the following message:

AssertionError: The length of pos_embed (197) doesn't match the expected shape (14*14+0). Please check the img_size argument.

However, when I do not use a pretrained model, the training proceeds without any messages.

These are the versions I am using on my system:

torch version: 2.0.1 True
torchvision version: 0.15.2
mmpose version: 1.1.0
cuda version: 11.8
compiler information: GCC 9.3

Is there any way to train a pretrained ViTPose model without getting this message?

Thank you very much for your assistance.

**Comment by 52THANOS** [comment]
i got the same issue

**Comment by Ben-Louis** [comment]
Sorry for the late reply. This issue has been resolved in the latest version. Please install mmpose 1.2.0 and use the config from that version. Additionally, make sure you use mmpretrain>=1.0.0.

**Reply by amartincirera** [reply]
Thank you so much!
